src/main/java/mate/academy/config/FieldMatch.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.config;
```

Comment:
move validation related classes to separate class.

it shouldn't be stored in config.